src/java/org/apache/cassandra/db/filter/ColumnFilter.java

```diff
@@ -325,50 +375,51 @@ public boolean fetchedCellIsQueried(ColumnMetadata column, CellPath path)
      * @return the created tester or {@code null} if all the cells from the provided column
      * are queried.
      */
-    public Tester newTester(ColumnMetadata column)
-    {
-        if (subSelections == null || !column.isComplex())
-            return null;
-
-        SortedSet<ColumnSubselection> s = subSelections.get(column.name);
-        if (s.isEmpty())
-            return null;
+    public abstract Tester newTester(ColumnMetadata column);
```

Comment:
Can we mark the method as `Nullable`?
```suggestion
    @Nullable
    public abstract Tester newTester(ColumnMetadata column);
```